src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -537,89 +593,92 @@ void func_80B14E28(EnTa* this, PlayState* play) {
     Play_CameraSetAtEye(play, this->subCamId, &subCamAt, &subCamEye);
 }
 
-void func_80B14EDC(EnTa* this, PlayState* play) {
+void EnTa_RemoveFloorCamera(EnTa* this, PlayState* play) {
     Play_ChangeCameraStatus(play, this->returnToCamId, CAM_STAT_ACTIVE);
     Play_ClearCamera(play, this->subCamId);
 }
 
-void func_80B14F20(EnTa* this, EnTaActionFunc arg1) {
-    func_80B13AA0(this, arg1, func_80B16854);
-    this->eyeIndex = 2;
+void EnTa_SetActionFuncWithBackToSleepAnimation(EnTa* this, EnTaActionFunc actionFunc) {
+    EnTa_SetupAction(this, actionFunc, EnTa_AnimSitSleeping);
+    this->eyeIndex = TALON_EYE_INDEX_CLOSED;
```

Comment:
maybe for these two we can say `EnTa_SetupActionWithSleepAnim`, and `EnTa_SetupActionWithWakeUpAnim`
this behaves very much like a normal "setup action" wrapper we see in other actors, so I dont think we need to say we are "setting the action func" in the name directly